docs/Protocol/publishBlockStream.md

```diff
@@ -0,0 +1,164 @@
+# Block Node Connect Protocol for `publishBlockStream`
+## Definitions
+
+Block Node
+: A software system intended to store and process a Block Stream.  The API for
+  a Block Node is defined in HIP 1056, among others.
+
+Block Number
+: A monotonically increasing number assigned by consensus to each block produced
+  by the network.
+
+Publisher
+: An entity publishing blocks to a Block Node via the `publishBlockStream` API
+  This is typically a Consensus Node or another Block Node.
+
+Subscriber
+: An entity that subscribes to a verified or unverified Block Stream from a
+  Block Node.
+
+Verified Block
+: A verified block is a block for which a Block Proof is received and for which
+  the TSS signature of the network ledger ID is valid.
```

Comment:
```suggestion
`Block Node`: A software system intended to store and process a Block Stream. The API for
  a Block Node is defined in `HIP-1056`, among others.

`Block Number`: A monotonically increasing number assigned by consensus to each block produced
  by the network.

`Publisher`: An entity publishing blocks to a Block Node via the `publishBlockStream` API. This is typically a Consensus Node or another Block Node.

`Subscriber`: An entity that subscribes to a verified or unverified Block Stream from a
  Block Node.

`Verified Block`: A verified block is a block for which a Block Proof is received and for which
  the TSS signature of the network ledger ID is valid.
```

Reply:
This is a definition list (github doesn't support them properly, unfortunately).
The syntax is markdown syntax for a definition list, not a mistake.

Would this read better using HTML definition list syntax?

Reply:
I think it did, thank you 👍